pyproject.toml

```diff
@@ -9,6 +9,10 @@ readme = "README.md"
 python = "^3.9, <3.12"  # pyinstaller requires Python <3.12
 grobid-tei-xml = "^0.1.3"
 grobid-client-python = "^0.0.5"
+sentence-transformers = "^2.2.2"
+lancedb = "^0.1.3"
```

Comment:
We can remove these from this commit, I believe?

Reply:
Yep. I think this only ended up here because other PRs got merged.